api/src/api/userRouter.test.ts

```diff
@@ -390,83 +390,6 @@ describe("userRouter", () => {
           },
         });
       });
-
-      it("does not add a blank signer if the legal structure is a corp", async () => {
```

Comment:
why are we deleting these tests?

Reply:
or i guess, why are we deleting this functionality?

Reply:
moved it to a simple useEffect hook in the signer component,